src/util/irep_ids.txt

```diff
@@ -800,4 +800,5 @@ cprover_string_to_char_array_func
 cprover_string_to_lower_case_func
 cprover_string_to_upper_case_func
 cprover_string_trim_func
-cprover_string_value_of_func
\ No newline at end of file
+cprover_string_value_of_func
+handler
```

Comment:
It's introduced in #506. That's where the exception handling instrumentation is being generated.